stablehlo/tests/interpret_exponential.mlir

```diff
@@ -0,0 +1,28 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: exponential_op_test_i64
+func.func @exponential_op_test_i64() -> tensor<5xf64> {
+  %operand = stablehlo.constant dense<[0.0, 1.0, -2.3, 3.1415926535897931, 0x7FF0000000000000]> : tensor<5xf64>
```

Comment:
What's our convention for the number of values in these tests? The spec says "As an exception to this rule, floating-point exceptions in StableHLO programs have well-defined behavior", so does this mean that we need to test all floating-point corner cases of numerical ops?

Reply:
The interpreter test input selection is solely based on the coverage of the implementation.  In this case the example we have in the spec should be good enough to fully cover our implementation. 

I agree that there are many interesting cases, but the current scope is to stick to basic ones providing good coverage.

Reply:
Sure, resolved by remove the 0x7FF0000000000000 and keep the rest.